modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -15,15 +15,71 @@ toc::[]
+
+Usually Space Managers are for example project leaders or teachers. 
+All in all, Spaces make life easier for admins by giving Space Managers the ability to self-service.
+
+==== ownCloud Web
+The new design of ownCloud Web comes with a brand new look. But our actual goal is to make owncloud accessible to all people from children to experts. Technically ownCloud Web is build with VueJS as dedicated client, communicating via well-formed APIs with the Infinite Scale server
+Customizsation: ownCloud Web can get customized according to your corporate identity via config values.
+Please find the full list of changes here: https://github.com/owncloud/web/releases/tag/v6.0.0
+
+==== Collaborative Editing
+
+Editing with Microsoft Online Office 365,  Collabora Office and OnlyOffice 
+
+Infinite Scale comes with a completely ramped up WOPI protocol server that reliably empowers collaborative office document editing with multi users. It supports Microsoft Online Office 365,  Collabora Office and OnlyOffice out of the box, and is ready to support more collaborative editors in the near future. With the focus of seamless collaboration, files are locked when opened in an online editor to visualize that to other users.
+
+==== Links
+
+In Infinite Scale we dramatically simplified sharing by links. First of all: You can get a link within a single click in ownCloud Web. Second: You can share one and the same link with internal and external people at once and if you change your mind, you can restict access for example only to internal people of your organisation. This feature was known as "private link" in ownCloud 10 and is still available, but via the "internal" selection of a link.
+
+==== Cloud Native Deployments
+
+Infinite Scale is a platform built of independent microservices that communicate over the highly efficient GRPC  protocol internally and provide an HTTP API externally. That gives the flexibility for easy extension, integration and deployment of the entire platform.
+
+For the sake of efficiency, Infinite Scale reduces dependencies consequently and works without a traditional database in the core - that saves administrational efforts, hardware resources and energy.
+
+One of the biggest benefits of Infinite Scale is the variety of deployment models that make the rollout of the platform really easy and independent of the underlying  infrastructure as far as possible.
+
+Infinite Scale is delivered as a so called "single binary" - which is a self contained, complete and ready to start compilation of all microservices that are needed to start Infinite Scale really simple. Another featured deployment method are docker images and docker-compose files. For setups with scaling requirements all  Infinite Scale microservices can be deployed and scaled separately on cloud infra like Kubernetes.
```

Comment:
```suggestion
Infinite Scale is delivered as a so called "single binary" - which is a self contained, complete and ready to start compilation of all microservices that are needed to start Infinite Scale really simple. Another featured deployment method are docker images and docker-compose files. For setups with scaling requirements all Infinite Scale microservices can be deployed and scaled separately on cloud infra like Kubernetes.
```

Reply:
Kubernetes is not cloud infrastructure, but a container orchestration system (which then spawns your cloud infrastructure). I'd rather say `[...] and scaled separately with orchestration systems like Kubernetes`